### ISSUE TITLE
[MDCSnackbar] Cannot access MDCSnackbar Accessibility Identifier

### ISSUE THREAD

**Issue by lane-faison** [material-components/material-components-ios#9259]
When running an XCTest that involves waiting for a MDCSnackbar to appear, the MDCSnackbar's label cannot be located by its default accessibility identifier (`"MDCSnackbarMessageTitleAutomationIdentifier"`). Also, the MDCSnackbar remains "Not Accessibility Element" according to the Xcode Accessibility Inspector, even after telling the snackbar's manager to make it accessible (`MDCSnackbarManager.setIsAccessibilityElement(true)`)

## Reproduction steps

Prerequisite: Configure your MDCSnackbarManager to `setIsAccessibilityElement` to `true`
1. On your simulator, perform any action that causes an MDCSnackbar to appear
2. Before the snackbar disappears, click the `Debug View Hierarchy` in Xcode
3. Inspect the MDCSnackbar and see that it is still "Not Accessibility Element"

## Expected behavior

- MDCSnackbar should be accessible and its label should be discoverable by an XCTest using `app.otherElements["MDCSnackbarMessageTitleAutomationIdentifier"]`

## Actual behavior

- MDCSnackbar is not accessible and its label cannot be located by an XCTest using `app.otherElements["MDCSnackbarMessageTitleAutomationIdentifier"]`

## Platform (please complete the following information)

 - Device: [e.g. iPhone11pro]
 - OS: [e.g. iOS13.2.2]

**Comment by randallli** [comment]
The labels are nil by default I thought.

In our unit tests we are successfully retrieving the accessibilityLabel.
https://github.com/material-components/material-components-ios/blob/develop/components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m#L121

@yar to confirm.
